indico/htdocs/js/indico/jquery/itempicker.js

```diff
@@ -29,8 +29,11 @@
         },
 
         _create: function() {
-            var items = this.options.items || this.element.data('items') || [];
-            var qtipContent = this._createQtipContent(items);
+            this.selectedItem = null;
+            this.previouslySelectedItem = null;
```

Comment:
Should this really be stored on the object? Sounds more like something to pass around to the callback - afterwards it shouldn't be needed anymore